bindings/go/plugin/manager/registries/plugins/wait_for_plugin.go

```diff
@@ -83,8 +84,18 @@ func getPluginLocation(ctx context.Context, plugin *types.Plugin) (string, error
 	timeoutCtx, cancel := context.WithTimeout(ctx, connectionTimeout)
 	defer cancel()
 
+	var reader io.Reader
+	switch {
+	case plugin.Stdout != nil && plugin.Stderr != nil:
+		reader = io.MultiReader(plugin.Stdout, plugin.Stderr)
+	case plugin.Stdout != nil:
+		reader = plugin.Stdout
+	case plugin.Stderr != nil:
+		reader = plugin.Stderr
+	}
```

Comment:
I'm unsure about this. The reason I didn't add scanning stderr here, is that it could potentially pollute the output and jeopardize the reading of a single line which should be there normally.

That said, maybe this is fine, since we read all lines and we just skip it if we couldn't parse it.

Reply:
It's fine, let's leave it and see. I _think_ the parsing will save this if anything pollutes stderr.